packages/web-twig/src/Resources/components/Collapse/Collapse.twig

```diff
@@ -0,0 +1,26 @@
+{# API #}
```

Comment:
There is missing `{%- set props = props | default([]) -%}` which was fixed in https://github.com/lmc-eu/spirit-design-system/pull/499